File: pset6/mario/less/mario.py
from cs50 import get_int

# get pyramid height from user
while True:
    height = get_int("Height of pyramid? ")
    # keep asking until valid input is accepted
    if height >= 1 and height <= 8:
        break
    else:
        continue
    

for i in range(0, height):
    # print spaces for flipped pyramid
    for j in range(1, height - i):
        print(end=" ")
        
    # print hashes for each row
    for hashes in range(0, i + 1):
        print("#", end="")

    # print new line after pyramid is printed
    print("")

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //initialize variable
    int n;
    //ask for height from user until given a number between 1-8 
    do
    {

        n = get_int("Height of pyramid? ");

    }
    while (n < 1 || n > 8);

    //print first half of pyramid with spaces only
    for (int x = 0; x < n; x++)
    {
        for (int s = n - x; s > 1; s--)
        {

            printf(" ");

        }
        //print first half of pyramid with hashes
        for (int h = n + x + 1; h > n; h--)
        {

            printf("#");

        }
        //print spaces inbetween the two pyramids
        printf("  ");
        //print second half of pyramid with hashes
        for (int y = 0; y < x; y++)
        {

            printf("#");

        }
        printf("#\n");
    }








}